### PR TITLE
add

### PR DESCRIPTION
**Pull request by arunt7767**
added a new theme
 name: "Neon Knights"
''' I'm just learning how to contribute to open source

### CODE REVIEW

**Comment by anuraghazra** [comment]
Theme preview (bot)  
      
      
title_color: <code>#00FF00</code> | icon_color: <code>#00FF00</code> | text_color: <code>#ffffff</code> | bg_color: <code>#000000</code>
      
      
Link: https://github-readme-stats.vercel.app/api?username=anuraghazra&title_color=00FF00&icon_color=00FF00&text_color=ffffff&bg_color=000000&show_icons=true

      
[](https://github-readme-stats.vercel.app/api?username=anuraghazra&title_color=00FF00&icon_color=00FF00&text_color=ffffff&bg_color=000000&show_icons=true)
      
      
  
> Hi thanks for the theme contribution, please read our theme contribution guidelines

  
> We are currently only accepting color combinations from any vscode theme or which has good color combination to minimize bloating the themes collection.

  
> Also note that if this theme is exclusively for your personal use then instead of adding it to our theme collection you can use card [customization options](https://github.com/anuraghazra/github-readme-stats#customization) 
  
> Read our [contribution guidelines](https://github.com/anuraghazra/github-readme-stats/blob/master/CONTRIBUTING.md) for more info

**Comment by arunt7767** [comment]
#first

**Comment by arunt7767** [comment]
ok 👍

File: themes/index.js [hunk 1]
```diff
@@ -294,6 +294,12 @@ const themes = {
     text_color: "FFFFFF",
     bg_color: "2C2F33",
   },
+  Neon Knights : {
```
Comment (rickstaa) [inline themes/index.js:297]: Invalid json syntax.

**Review by rickstaa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**